common/test/acceptance/tests/lms/test_lms_courseware.py

```diff
@@ -156,7 +157,7 @@ def setUp(self):
             self.course_info['run'], self.course_info['display_name']
         )
         course_fix.add_advanced_settings({
-            "enable_proctored_exams": {"value": "true"}
+            "enable_special_exams": {"value": "true"}
```

Comment:
This should not be changed, it's a course-specific advanced setting. The installation-wide feature flag is ENABLE_SPECIAL_EXAMS.

https://github.com/edx/edx-platform/blob/master/cms/static/js/views/modals/course_outline_modals.js#L51